src/tools/tidy/src/triagebot.rs

```diff
@@ -0,0 +1,24 @@
+use crate::walk::{filter_dirs, walk};
+use serde::Deserialize;
+use std::path::Path;
+use toml::Value;
+
+#[derive(Deserialize)]
+struct Config {}
+
+pub fn check(path: &Path, bad: &mut bool) {
+    walk(path, &mut |path| filter_dirs(path), &mut |entry, contents| {
+        let file = entry.path();
+        let filename = file.file_name().unwrap();
+        if filename != "triagebot.toml" {
```

Comment:
Maybe only check the top-level `triagebot.toml` instead of trying to find it in all subdirectories?

Reply:
Sure, I'll fix it. I had searched the repo for the `triagebot.toml` files and saw multiple, hence I thought maybe I should do a walk. Is there a reason we don't check those toml files?

Reply:
Only the top-level one is actually used by triagebot afaik. Several subdirectories of `src/tools` are submodules or subtrees. Their copy is only used for the separate repo from which they are pulled in. For example `src/tools/miri` is a subtree pulled from https://github.com/rust-lang/miri/, so the https://github.com/rust-lang/miri/ repo has a `triagebot.toml` file which is used when interacting with this repo, but it is also copied into `src/tools/miri/triagebot.toml` as part of `git subtree` syncs, as everything is copied.